bertopic/_bertopic.py

```diff
@@ -797,7 +803,10 @@ def _extract_topics(self, documents: pd.DataFrame):
         Returns:
             c_tf_idf: The resulting matrix giving a value (importance score) for each word per topic
         """
-        documents_per_topic = documents.groupby(['Topic'], as_index=False).agg({'Document': ' '.join})
+        # documents_per_topic = documents.groupby(['Topic'], as_index=False).agg({'Document': ' '.join})
+        documents_per_topic = documents.groupby(['Topic'], as_index=False).agg(
+            {'Document': lambda topic: [token for doc in topic for token in doc]})
```

Comment:
When grouped by `Topic`, does it produce `list of list of list of tokens`?

Reply:
Done!